HW3/Task_1_06_03_23.py

```diff
@@ -0,0 +1,44 @@
+
+# Завдання 1
+print("Завдання 1")
+print()
```

Comment:
nit: `\n` can be used instead of`print()`
```suggestion
print("Завдання 1\n")
```